src/classes/FoundBugs.js

```diff
@@ -276,6 +276,16 @@ class FoundBugs {
 		return apps_arr;
 	}
 
+	getAppsById(tab_id) {
```

Comment:
Please add a documentation comment for this function.

Reply:
Added